### ISSUE TITLE
Refreshing the Timeline page takes you back to the New Tab page

### ISSUE THREAD

**Issue by pdehaan** [mozilla/activity-stream#1573]
![add-ons_manager](https://cloud.githubusercontent.com/assets/557895/19453661/12248ee4-946b-11e6-98ae-fe10130dbe2a.png)

### Steps to reproduce:

1. Install 1.1.5 beta.
1. Open new Tab.
1. Scroll to the bottom and then click the [See all activity] button in the lower-left.
1. On the Timeline page, hit the Refresh button in the address bar.

### Actual results:
Page is refreshed to the New Tab page, not the current Timeline page.

![activity_stream](https://cloud.githubusercontent.com/assets/557895/19453798/92357648-946b-11e6-86d5-d14d7160ebe6.png)
**Before:** Timeline page.

![new_tab](https://cloud.githubusercontent.com/assets/557895/19453817/a992ef3c-946b-11e6-831d-df6a672a7350.png)
**After:** New Tab page.

### Expected results:
Refresh should refresh the current page, not the previous New Tab page.

**Comment by Mardak** [comment]
Duplicate of #1333.
